packages/vite-plugin-svelte/src/utils/log.js

```diff
@@ -262,7 +262,10 @@ export function isDebugNamespaceEnabled(namespace) {
 }
 
 export function logSvelte5Warning() {
-	const notice = `You are using Svelte ${VERSION}. Svelte 5 support is experimental, breaking changes can occur in any release until this notice is removed.`;
-	const wip = ['svelte-inspector is disabled until dev mode implements node to code mapping'];
-	log.warn(`${notice}\nwork in progress:\n - ${wip.join('\n - ')}\n`);
+	log.warn(
+		`You are using Svelte ${VERSION} with vite-plugin-svelte@3. Active Svelte 5 support has moved to vite-plugin-svelte@4.
+			To receive bugfixes and new features update your devDependencies to "@sveltejs/vite-plugin-svelte": "^4.0.0-next.6" and install.
+			For framework integrations that depend on it (e.g. astro), you might have to add an override:
+			"overrides": {"@sveltejs/vite-plugin-svelte": "^4.0.0-next.6"}`.replace(/\n\s*/gm, '\n\t')
```

Comment:
```suggestion
		`You are using Svelte ${VERSION} with vite-plugin-svelte@3. Active Svelte 5 support has moved to vite-plugin-svelte@4.
			To receive bug fixes and new features update your devDependencies to "@sveltejs/vite-plugin-svelte": "^4.0.0-next.6" and install.
			For framework integrations that depend on it, you might have to add an override:
			"overrides": {"@sveltejs/vite-plugin-svelte": "^4.0.0-next.6"}`.replace(/\n\s*/gm, '\n\t')
```

I don't think we need to mention astro here to avoid preferential support 👀